pkgs/development/tools/loganalyzer/default.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, qtbase
+, qtsvg
+, wrapQtAppsHook
+}:
+
+let
+  pname = "loganalyzer";
+  appname = "LogAnalyzer";
+  version = "23.5.1";
+in
+stdenv.mkDerivation {
+  inherit pname appname version;
+
+  src = fetchFromGitHub {
+    owner = "pbek";
+    repo = "loganalyzer";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-k9hOGI/TmiftwhSHQEh3ZVV8kkMSs1yKejqHelFSQJ4=";
+  };
+
+  buildInputs = [
+    qtbase
+    qtsvg
+  ];
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+  ];
+
+  sourceRoot = "source/src";
+
+  buildPhase = ''
+    runHook preBuild
+
+    qmake ${appname}.pro CONFIG+=release PREFIX=/
+    make
+
+    runHook postBuild
+  '';
+
+  installFlags = [ "INSTALL_ROOT=$(out)" ];
+
+  postInstall =
+  # Create a lowercase symlink for Linux
+  lib.optionalString stdenv.isLinux ''
+    ln -s $out/bin/${appname} $out/bin/${pname}
+  '';
```

```suggestion
  postInstall = ''
    ln -s $out/bin/LogAnalyzer $out/bin/loganalyzer
  '';
```
or does the path not exist on darwin?

---

I guess it exists...